src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -14,6 +18,37 @@ class Tribe__Tickets__Repositories__Attendee__RSVP extends Tribe__Tickets__Atten
 	 */
 	protected $key_name = 'rsvp';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.rsvp' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it PHP <7.0
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 6000d3a5574abf9b7198daedd21702ad</sub>